tools/apidiff/merger.cs

```diff
@@ -13,6 +13,14 @@ static string GetVersion (string line)
 		return line.Substring (start, end - start);
 	}
 
+	static string DestinationPath = null;
```

Comment:
No need for the static var. We can pass that as an optional arg in the Process call:
```csharp
public static void Process (string platform, string path, string os, string? destinationDirectory = null);
```
That make the need to have the static var and the extra method overload to not be needed. And you can simply do the example same call you have in:
```csharp
var filePath = destinationDirectory is not null ? Path.Combine (destinationDirectory, filename) : filename;
```
That also means that the code in the main method can be simpler.